### ISSUE TITLE
Exception for doctor role

### ISSUE THREAD

**Issue by psykzz** [werewolves-and-nobody/werewolves#11]
``` Listening.
  werewolves:game Adding new player to pool: a +0ms
  werewolves:game Adding new player to pool: b +2s
  werewolves:game Doing action #0: setNightAction +3ms
  werewolves:game Doing action #1: werewolvesAction +1ms
  werewolves:game Notifying werewolves, time to eat +0ms
  werewolves:game Eating: a +8s
  werewolves:game Doing action #2: doctorsAction +2ms
Missing error handler on `socket`.
ReferenceError: self is not defined
    at Game.notifyRfa (C:\Users\PsyKzz\Documents\GitHub\werewolves\game.js:157:25)
    at Game.doctorsAction (C:\Users\PsyKzz\Documents\GitHub\werewolves\game.js:204:8)
    at Game.doNextAction (C:\Users\PsyKzz\Documents\GitHub\werewolves\game.js:151:19)
    at C:\Users\PsyKzz\Documents\GitHub\werewolves\game.js:254:12
    at C:\Users\PsyKzz\Documents\GitHub\werewolves\node_modules\async\dist\async.js:484:16
    at C:\Users\PsyKzz\Documents\GitHub\werewolves\node_modules\async\dist\async.js:1161:9
    at C:\Users\PsyKzz\Documents\GitHub\werewolves\node_modules\async\dist\async.js:484:16
    at iteratorCallback (C:\Users\PsyKzz\Documents\GitHub\werewolves\node_modules\async\dist\async.js:1084:13)
    at C:\Users\PsyKzz\Documents\GitHub\werewolves\node_modules\async\dist\async.js:988:16
    at C:\Users\PsyKzz\Documents\GitHub\werewolves\node_modules\async\dist\async.js:1158:13
```

**Comment by psykzz** [comment]
This only applies to the doctor-role branch

**Comment by psykzz** [comment]
fixed in #13
